plugins/dolt/scripts/flytekit_install_dolt.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# Fetches and install Dolt. To be invoked by the Dockerfile
+
+# echo commands to the terminal output
+set -eox pipefail
+
+# Install Dolt
+
+apt-get update -y \
+    && apt-get install curl \
+    && sudo bash -c 'curl -L https://github.com/dolthub/dolt/releases/latest/download/install.sh | sudo bash'
```

Comment:
@wild-endeavor Something i was confused about before -- will CI run this script to install Dolt?

Reply:
CI will run this https://github.com/flyteorg/flytekit/blob/master/.github/workflows/pythonbuild.yml#L37 so as long as it gets picked up from that we should be good.

I'm also thinking we'll need to move plugins into a separate repo sooner rather than later.